collectors/freeipmi.plugin/multi_metadata.yaml

```diff
@@ -141,6 +141,7 @@ modules:
     meta:
       <<: *meta
       monitored_instance:
+        <<: *monitored_instance
```

Comment:
is this really needed? Just copy/paste/change monitored_instance

Reply:
it might be overkill for one virtual integration only, I don't think it is an issue here though

Reply:
Do you mean overkill to use anchors here? I agree, hence the comment